MovingBlocks/TerasologyLauncher
# test: Fix Windows tests

### Contains

Updated tests for `TestFileUtils` correctly handling Windows ACL permissions.
Fixes #595 

### How to test

- Clone on Windows, run `./gradlew test`. All test should now pass.
- Clone on Linux, run `./gradlew test`. All test should pass as they did before.

### Outstanding before merging

I've also updated the IntelliJ part of `.gitignore`, as I use IntelliJ on daily basis for work and I have few plugins that were not covered in the original one. I usually go for the one from gitignore.io, proved quite accurate and useful by the years.
Consider if useful or not.
